plugins/statuslines/lualine.nix

```diff
@@ -1,260 +1,281 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.lualine;
+  inherit (lib.nixvim) defaultNullOpts mkSettingsRenamedOptionModules;
+  inherit (lib) types mkOption;
 
+  # TODO: do we really need this
   mkSeparatorsOption =
     {
       leftDefault ? " ",
       rightDefault ? " ",
     }:
     {
-      left = helpers.defaultNullOpts.mkStr leftDefault "Left separator";
-      right = helpers.defaultNullOpts.mkStr rightDefault "Right separator";
+      left = defaultNullOpts.mkStr leftDefault "Left separator";
+      right = defaultNullOpts.mkStr rightDefault "Right separator";
     };
 
   mkComponentOptions =
```

Comment:
Let's add a comment describing the deprecation, when it was added and what it does.